packages/flutter_tools/test/general.shard/web/web_expression_compiler_test.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:dwds/dwds.dart';
+import 'package:dwds/src/services/expression_compiler.dart';
```

Comment:
What is getting loaded from here? This is less than ideal.

Reply:
Apparently it is completely unneeded - I guess it got added automatically:) Will remove.

Reply:
done, thanks!